notebooks/1-lightcurves/TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1265 @@
+{
```

Comment:
<p>The "other common uses of time series data" are not covered in this notebook. Should either mention them at the end (and provide links to existing notebooks, if any), or remove this bullet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>added more info to the end, but this notebook was not supposed to include actual examples of this so I updated the text in bullet to reflect that. </p>